doc/proposals/olm-cmd-workflows.md

```diff
@@ -0,0 +1,120 @@
+---
+title: README describing new command workflows for olm enabled operators
+authors:
+  - "@varshaprasad96"
+reviewers:
+  - "@jlanford"
+  - "@dmesser"
+  - "@estroz"
+approvers:
+  - "@jlanford"
+  - "@dmesser"
+  - "@estroz"
+creation-date: 2020-04-09
+last-updated: 2020-04-09
+status: provisional
+---
+
+# README describing new command workflows for olm enabled operators
```

Comment:
when referring to the product or project name I think we should use capitals. olm -> OLM